jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/ExtendedVariantWidget.js

```diff
@@ -118,26 +118,31 @@ export default jbrowse => {
         const displayJSX = []
         for (let i = 0; i < propertyJSX.length; i++){
             displayJSX.push(
-                <BaseCard title={displays[i].name}>
-                    {propertyJSX[i]}
-                </BaseCard>
+                    <BaseCard title={displays[i].name}>
+                        {propertyJSX[i]}
+                    </BaseCard>
             )
         }
         return displayJSX
     }
 
     function makeChart(samples, feat, classes){
+        // Abort if there are no samples
+        if (Object.keys(samples).length === 0) {
```

Comment:
See above.

Reply:
@jctrotter I'm fixing a bug in your code. no question this falls within the scope of code cleanup.